src/main/java/com/serhiiklymchuk/gmailpet/config/AsyncConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.serhiiklymchuk.gmailpet.config;
+
+import com.serhiiklymchuk.gmailpet.domain.Message;
+import com.serhiiklymchuk.gmailpet.dto.MessageFormDto;
+import com.serhiiklymchuk.gmailpet.repository.MessageRepository;
+import org.springframework.aop.interceptor.AsyncUncaughtExceptionHandler;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.AsyncConfigurer;
+
+import java.time.LocalDateTime;
+
+@Configuration
+public class AsyncConfig implements AsyncConfigurer {
+
+    private static final Long GMAIL_SUPPORT_ID = 1L;
+
+    private final MessageRepository messageRepository;
```

Comment:
When I Autowire service , async stopes working.
Maybe, because, I call async method `createMessage` inside Async...Handler and it stucks in a loop.
And `method.invoke()` also is not possible cause I need instance of service to call it.